library/tasks/HISAT2.wdl

```diff
@@ -71,8 +72,8 @@ task HISAT2PairedEnd {
       --seed 12345 \
       -k 10 \
       --secondary \
-      -p ${cpu} -S ${output_basename}.sam
-    samtools sort -@ ${cpu} -O bam -o "${output_basename}.bam" "${output_basename}.sam"
+      -p ${cpu} -S >(samtools view -1 -h -o ${output_basename}_unsorted.bam)
```

Comment:
Instead of hard coding the parallelism, maybe use something like `$(nproc)`. I don't think executors are necessarily required to honor the `runtime.cpu` value.

Reply:
Just a side note that `nproc` might not exist in some docker images.

Reply:
I'll create a ticket for this to be addressed in a future PR.

Reply:
https://app.zenhub.com/workspace/o/humancellatlas/secondary-analysis/issues/363